src/main/g8/src/main/scala/$package__packaged$/Server.scala

```diff
@@ -4,22 +4,23 @@ import java.util.concurrent.{ExecutorService, Executors}
 
 import scala.util.Properties.envOrNone
 
-import scalaz.concurrent.Task
+import fs2.Task
 
-import org.http4s.server.{Server, ServerApp}
+import org.http4s.server.Server
+import org.http4s.util.StreamApp
 import org.http4s.server.blaze.BlazeBuilder
 
 
-object BlazeExample extends ServerApp {
+object BlazeExample extends StreamApp {
 
   val port : Int              = envOrNone("HTTP_PORT") map (_.toInt) getOrElse 8080
   val ip   : String           = "0.0.0.0"
   val pool : ExecutorService  = Executors.newCachedThreadPool()
 
-  override def server(args: List[String]): Task[Server] =
+  def main(args: List[String]) =
```

Comment:
I agree.  There's also talk of renaming `main`, but that's a discussion for the other repo. :)